packages/errors/src/codes.ts

```diff
@@ -51,6 +51,12 @@ export const SOLANA_ERROR__CODECS_FIXED_NULLABLE_WITH_VARIABLE_SIZE_PREFIX = 44
 export const SOLANA_ERROR__CODECS_CODEC_REQUIRES_FIXED_SIZE = 45 as const;
 export const SOLANA_ERROR__CODECS_NUMBER_OUT_OF_RANGE = 46 as const;
 export const SOLANA_ERROR__CODECS_INVALID_STRING_FOR_BASE = 47 as const;
+export const SOLANA_ERROR__BLOCKHASH_STRING_LENGTH_OUT_OF_RANGE = 48 as const;
+export const SOLANA_ERROR__BLOCKHASH_BYTE_LENGTH_OUT_OF_RANGE = 49 as const;
+export const SOLANA_ERROR__LAMPORTS_OUT_OF_RANGE = 50 as const;
+export const SOLANA_ERROR__BIGINT_STRING_MALFORMED = 51 as const;
+export const SOLANA_ERROR__NUMBER_STRING_MALFORMED = 52 as const;
```

Comment:
nit: We may want to stay consistent with other existing error names such as `SOLANA_ERROR__MALFORMED_PROGRAM_DERIVED_ADDRESS` and use `MALFORMED` as a prefixed instead of a suffix.

Unless we prefer the suffix version, in which case we should probably update the other ones.

Reply:
I vote prefix!